addons/website_sale/views/templates.xml

```diff
@@ -212,7 +212,7 @@
             <t t-set="product_variant_id" t-value="product._get_first_possible_variant_id()"/>
             <input name="product_id" t-att-value="product_variant_id" type="hidden"/>
             <t t-if="product_variant_id">
-                <a href="#" role="button" class="btn btn-primary a-submit" aria-label="Shopping cart" title="Shopping cart">
+                <a href="#" role="button" class="btn btn-primary a-submit o_shopping_cart" aria-label="Shopping cart" title="Shopping cart">
```

Comment:
Could `o_shopping_cart` rather be a name? Like `o_products_grid_cart_button`. I find it confusing to use a class to select an element, as it usually also involves setting a CSS class.